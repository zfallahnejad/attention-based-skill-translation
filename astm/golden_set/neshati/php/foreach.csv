1491895,foreach
1338292,foreach
476,foreach
541091,foreach
1226894,foreach
1438393,foreach
862594,foreach
1003917,foreach
507674,foreach
42139,foreach
50079,foreach
250259,foreach
251311,foreach
696364,foreach
1570534,foreach
1529853,foreach
53114,foreach
1518087,foreach
171318,foreach
689579,foreach
113938,foreach
2364629,foreach
574632,foreach
138475,foreach
283366,foreach
822711,foreach
557846,foreach
208809,foreach
223386,foreach
3859027,foreach
298053,foreach
889949,foreach
995579,foreach
187606,foreach
697154,foreach
