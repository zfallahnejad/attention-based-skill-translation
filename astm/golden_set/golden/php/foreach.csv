1491895,53,31,0.6309523809523809
1338292,24,21,0.5333333333333333
476,22,28,0.44
541091,21,7,0.75
1226894,18,17,0.5142857142857142
1438393,16,13,0.5517241379310345
862594,15,6,0.7142857142857143
1003917,14,11,0.56
42139,13,14,0.48148148148148145
507674,13,15,0.4642857142857143
50079,12,9,0.5714285714285714
251311,11,7,0.6111111111111112
250259,11,7,0.6111111111111112
1570534,10,8,0.5555555555555556
1529853,10,6,0.625
696364,10,1,0.9090909090909091
53114,9,9,0.5
171318,9,7,0.5625
113938,9,5,0.6428571428571429
689579,9,4,0.6923076923076923
1518087,9,2,0.8181818181818182
574632,8,4,0.6666666666666666
138475,8,5,0.6153846153846154
2364629,8,8,0.5
283366,8,2,0.8
822711,8,0,1.0
298053,7,7,0.5
697154,7,0,1.0
3859027,7,4,0.6363636363636364
557846,7,7,0.5
889949,7,7,0.5
187606,7,4,0.6363636363636364
208809,7,4,0.6363636363636364
223386,7,7,0.5
995579,7,8,0.4666666666666667
